src/internal/index.ts

```diff
@@ -15,9 +15,9 @@ export { default as getUniV3Pool } from './getUniV3Pool'
 
 export { default as groupAndSumTokenAmounts } from './groupAndSumTokenAmounts'
 
-export { default as intentArgsToStrategyArgs } from './intentArgsToStrategyArgs'
```

Comment:
See my comment on the types renaming PR (https://github.com/brinktrade/brink-types/pull/27/files#diff-f373d4637e1a9247d14a4b89b4d4592f4ddb59ba8bca1101edcc4dcd01e4955fR241)

these will all need to be renamed to reflect that change